scripts/release/version_doc.py

```diff
@@ -0,0 +1,34 @@
+import yaml
+
+
+def table(list_of_dicts):
+    markdown_table = ""
+    markdown_header = '| ' + ' | '.join(map(str, list_of_dicts[0].keys())) + ' |'
+    markdown_header_separator = '|-----' * len(list_of_dicts[0].keys()) + '|'
+    markdown_table += markdown_header + '\n'
+    markdown_table += markdown_header_separator + '\n'
+    for row in list_of_dicts:
+        markdown_row = ""
+        for key, col in row.items():
```

Comment:
Ordering of items in dicts are not guaranteed, but constructing the table this way assumes each dict's keys are iterated accross in the same order.  We need to intelligently construct the table such that the keys are fetched property

Reply:
changed the input yaml to explicitly specify a col spec which we then leverage for indexing rows